components/button/button-subtle.js

```diff
@@ -30,19 +30,19 @@ export class D2LButtonSubtle extends D2LButtonMixin(LitElement) {
 	render() {
 		return html`
 			<button
-				aria-expanded="${this.ariaExpanded}"
-				aria-haspopup="${this.ariaHaspopup}"
-				aria-label="${this.ariaLabel}"
-				?autofocus="${this.autofocus}"
+				aria-expanded="${ifDefined(this.ariaExpanded)}"
+				aria-haspopup="${ifDefined(this.ariaHaspopup)}"
+				aria-label="${ifDefined(this.ariaLabel)}"
+				?autofocus="${ifDefined(this.autofocus)}"
```

Comment:
Interesting that it wouldn't automatically do this... I guess that's a major difference from Polymer? Maybe we should be keeping a list of this kind of thing?

Reply:
Yeah, maybe they don't want to make any assumptions about values and what to render, but in the case of `undefined` it may have been safe for them to handle?!?  I can easily see this being forgotten.